packages/router/src/grpc/view-protocol-server/fmd-parameters.test.ts

```diff
@@ -23,7 +23,9 @@ describe('FmdParameters request handler', () => {
       getFmdParams: vi.fn(),
     };
     mockServices = {
-      getWalletServices: vi.fn(() => Promise.resolve({ indexedDb: mockIndexedDb })),
+      getWalletServices: vi.fn(() =>
+        Promise.resolve({ indexedDb: mockIndexedDb }),
+      ) as MockServices['getWalletServices'],
```

Comment:
Given we have to build all these packages, type errors are now getting caught that went unchecked before

Reply:
#680